app/controllers/cases/index.js

```diff
@@ -25,18 +25,12 @@ export default class CasesIndexController extends Controller {
         type: 'string',
       },
     },
-    {
-      showArchived: {
-        type: 'boolean',
-      },
-    }
   ];
   sizeOptions = Object.freeze([5, 10, 20, 50, 100, 200]);
   page = 0;
   size = 20;
 
   sort = '-opened';
-  showArchived = false;
   @tracked isLoadingModel;
```

Comment:
This change was requested, but now the view is showing both unarchived and archived cases without a clear difference.
To the user, it seems like "delete case" action does nothing.
It would make more sense to never show archived cases or to show archived cases with a different css class (greyed out)
Because the feature "restore case" is still a thing that users use (recent support issue when it was broken) we still need to offer it in some way and you can't access it from search
Could we instead toggle showing all normal cases only / all archived cases only.
Ideally, I was thinking of an action menu inside the case route on individual cases but seems out of scope.

Reply:
In the search results rework, there is a visual indication that a case is archived. I'm not sure about the usefulness of having the ability to see archived and non-archived cases in the same view but I don't see why it should not be possible.
![image](https://user-images.githubusercontent.com/37635647/216944076-c763cc3f-f718-4878-8b5d-3838002ab554.png)

Reply:
I was referring to the datatable in the general case route (not search)
That table now shows both "deleted" and "not deleted", while previously it showed only all "not deleted.
You then had to trigger the "show deleted" button and the list would contain both (but not ordered or visually different).
So finding any specific archived case was just a no go.

With this suggested change, now you always see all cases no matter if you archive them or not which may lead users to conclude that "delete case" or "restore case" feature does nothing as the list stays the same.
Something needs to change there, but reworking it is out of scope.
So my suggestion would be to change the previous behavior "show deleted" to "show only deleted" so we can toggle between both lists (deleted and not deleted)
It would make it easier to find deleted cases (used but only seldom by users) and restore them.